### DISCUSSION TITLE
After clicking the Slack link, it says: There are no accounts in this workspace.

### DISCUSSION THREAD

**Discussion by LJTian** [sustainable-computing-io/kepler#1495] [Q&A]
I would like to join slack to ask questions that concern me. I don't know how to do this right now. Can you help me?

**Comment by dave-tucker** [comment]
I'm not sure which Slack link you are referring to, but we use the #kepler-project channel on the [CNCF Slack](https://slack.cncf.io).
If you're still having issues let me know and I'll see who we can track down to address this.

**Reply by LJTian** [reply]
Login prompt: No account.
Maybe I have to register through the link, but I don't know how to register.

**Reply by LJTian** [reply]
Successfully joined, thank you
